src/screens/Settings/ComplianceCertificates/Compliance.res

```diff
@@ -0,0 +1,79 @@
+module DownloadCertificateTile = {
+  @react.component
+  let make = (~header, ~onClick) => {
+    <div
+      className="flex flex-col bg-white pt-6 pl-6 pr-8 pb-8 justify-between gap-10 border border-jp-gray-border_gray rounded">
+      <div>
+        <p className="text-fs-16 font-semibold m-2"> {header->React.string} </p>
+      </div>
+      <Button
+        text="Download"
+        buttonSize={Medium}
+        buttonType={Primary}
+        rightIcon={FontAwesome("download-api-key")}
+        onClick
+      />
+    </div>
+  }
+}
+
+@react.component
+let make = () => {
+  let showToast = ToastState.useShowToast()
+  let fetchApi = AuthHooks.useApiFetcher()
+
+  let downloadPDF = () => {
+    let currentDate =
+      Date.now()
+      ->Js.Date.fromFloat
+      ->Date.toISOString
+      ->TimeZoneHook.formattedISOString("YYYY-MM-DD HH:mm:ss")
+
+    let downloadURL = Window.env.dssCertificateUrl->Option.getOr("")
+
+    //? - For localtesting this condn added
+    if downloadURL->LogicUtils.isNonEmptyString {
```

Comment:
can we remove this local testing condition?

Reply:
It can't be removed because it also ensures that the API isn't called when the env is empty.

Reply:
okay, then let's remove the comment

Reply:
I don't think we need to remove the comment because, in the local environment, there won't be a dssCertificateUrl, allowing the local user to proceed with just an error popup.